src/RepoAuditor/EntryPoint.py

```diff
@@ -165,7 +165,7 @@ def TypeInfoToString(
 
 
 # ----------------------------------------------------------------------
-def _VersionCallback(value: bool) -> None:
+def _VersionCallback(value: bool) -> None:  # noqa: FBT001

```

Comment:
It was not clear how and whether we should put much effort into fixing this, so I disabled the warning for that call.
See [ruff's docs on FBT001](https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/) for more context on that error